C/plugins/common/omf.cpp

```diff
@@ -319,8 +323,13 @@ uint32_t OMF::sendToServer(const vector<Reading *>& readings,
 	// Then get HTTPS POST ret code and return 0 to client on error
 	try
 	{
+		std::chrono::high_resolution_clock::time_point t1 = std::chrono::high_resolution_clock::now();
```

Comment:
I wonder how expensive it is to track the time?

Reply:
I don't think much, but I can remove it if you wish. Or I can find out the overhead.

Reply:
Actually, when we have sent out 5000 readings to Pi server, it takes around 700-1000 msec to get POST request serviced, so in that sense overhead of time calculation is insignificant.

Reply:
Tried it on my ubuntu VM, I got "1000 times duration calc took 213 usecs" and "100000 times duration calc took 17590 usecs". So it's around 175-213 nano-secs.

        std::chrono::high_resolution_clock::time_point t1 = std::chrono::high_resolution_clock::now();
	std::chrono::high_resolution_clock::time_point t2;
	int i;
	for (i=0; i<1000; i++)
		t2 = std::chrono::high_resolution_clock::now();
 	auto usecs = std::chrono::duration_cast<std::chrono::microseconds>( t2 - t1 ).count();
	logger->info("%d times duration calc took %lld usecs", i, usecs);

Reply:
OK, fine thanks.